hazelcast/src/main/java/com/hazelcast/client/impl/clientside/HazelcastClientInstanceImpl.java

```diff
@@ -167,7 +169,7 @@
     private final ClientExecutionServiceImpl executionService;
     private final ClientListenerServiceImpl listenerService;
     private final ClientTransactionManagerServiceImpl transactionManager;
-    private final NearCacheManager nearCacheManager;
+    private final Map<String, NearCacheManager> nearCacheManagers;
```

Comment:
Minor: Better to change type of `nearCacheManagers` to `ConcurrentMap` from `Map` to emphasize this map is subject to concurrent access.

Reply:
comment is addressed